### PR TITLE
feat: making call arguments collapsible

### LINKED ISSUE #567: Consider making call arguments collapsible
https://aescan.io/transactions/th_2KUovbF7qeofTsuxXfXkYkou7gUtzQGA2sioeoQquRv143Wyg2 😜

something like a white gradient with an arrow at the bottom would ease checking TXs with a ton of input like here. 👋

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #567 

## Demo

https://github.com/aeternity/aescan/assets/15363559/04707f93-e78d-4736-9216-bb257b27ab97

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] making call arguments collapsible

### CODE REVIEW

File: src/components/TransactionTypeTableContractCallTx.vue [hunk 1]
```diff
@@ -133,6 +149,14 @@ import TransactionTypeStatusLabel from '@/components/TransactionTypeStatusLabel'
 import { formatAePrice, formatAettosToAe, formatNullable } from '@/utils/format'
 import AppChip from '@/components/AppChip'
 
+const isCollapsed = ref(true)
+const isArgumentsLong = computed(() => JSON.stringify(props.transactionData.arguments).length > 300)
+const shortArguments = computed(() => isArgumentsLong.value ? JSON.stringify(props.transactionData.arguments).substr(0, 300) + '...' : props.transactionData.arguments)
+
```
Comment (janmichek) [inline src/components/TransactionTypeTableContractCallTx.vue:155]: threshold of 300 characters

**Comment by Liubov-crypto** [comment]
https://pr-688-aescan-testnet.stg.service.aepps.com/ - was not working for me, I got 404 error. 

@janmichek Could you please provide me some hash for contract call? I tried different types of Txs but was not able to find a Tx with extensive argument to check it properly.

**Comment by janmichek** [comment]
> https://pr-688-aescan-testnet.stg.service.aepps.com/ - was not working for me, I got 404 error.
> 
> @janmichek Could you please provide me some hash for contract call? I tried different types of Txs but was not able to find a Tx with extensive argument to check it properly.

It's mentioned in the initial issue .)

**Review by michele-franchi — APPROVED** [review]
Nice addition

**Comment by nikita-fuchs** [comment]
awesome !

**Pull request merged**